pkgs/tools/package-management/appimage-run/default.nix

```
@@ -66,6 +66,7 @@ buildFHSUserEnv {
     dbus-glib
     libav
     atk
+    at_spi2_atk
```

```suggestion
    at-spi2-atk
```
That's actually an alias to that if I remember correctly.